docs/src/main/sphinx/ext/redirects.py

```diff
@@ -0,0 +1,87 @@
+#
+#   sphinxcontrib.redirects
+#   ~~~~~~~~~~~~~~~~~~~~~~~
+#   Generate redirects for moved pages when using the HTML builder.
+#   :copyright: Copyright 2017 by Stephen Finucane <stephen@that.guru>
+#   :license: BSD, see LICENSE for details.
+#   https://github.com/sphinx-contrib/redirects/blob/master/LICENSE
+#
+#   Version 0.1.0 modified April 10, 2023 for Trino use
+#   See https://github.com/sphinx-contrib/redirects/pull/6
```

Comment:
This PR is already merged. Why not wait for the next release?

Reply:
We asked for a release .. seems like they team is not responding at this stage.. hence we pulled the extension in for now.. once a release becomes available we will switch bak to the official extension